docs/UserGuide.md

```diff
@@ -181,7 +185,7 @@ The table below summarises the add command, format and examples for each contact
 
 * Name is case-insensitive but space-sensitive.<br>
 
-* Email must in format {example}@{domain} or {example}@{domain}.{extension}
+* Email must in format {example}@{domain} or {example}@{domain}.{extension} <br>
 
 * Phone number must be in format {Number} and at least 3 digits long.<br>
```

Comment:
grammar error, "Phone number must be in **the** format..."

Reply:
changed